deepchem/metalearning/__init__.py

```diff
@@ -1,5 +1,68 @@
 # flake8: noqa
+import logging
+
+logger = logging.getLogger(__name__)
 try:
-    from deepchem.metalearning.maml import MAML, MetaLearner
-except ModuleNotFoundError:
-    pass
+    from deepchem.metalearning.maml import MAML
+    from deepchem.metalearning import MAML
+except ModuleNotFoundError as e:
+    logger.warning(
+        f'Skipped loading some PyTorch models, missing a dependency. {e}')
+
+
+class MetaLearner(object):
```

Comment:
This will cause a circular import issue. Move this into its own file metalearner.py and importt it before the other files up top

Reply:
It's not a circular import because we are not inheriting from it.